pkg/vdr/web/resolver.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package web
+
+import (
+	"fmt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+	"io/ioutil"
+	"net/http"
+)
+
+// Read resolves a did:web did
+func (v *VDR) Read(didID string, opts ...vdr.ResolveOpts) (*did.Doc, error) {
+
+	// TODO actually use the ResolveOpts
+
+	// parse doc url from did
+	url, err := parseDIDWeb(didID)
+	if err != nil {
+		return nil, fmt.Errorf("error resolving did:web did --> could not resolve did:web did --> %w", err)
+	}
+
+	// send get request to retrieve did doc
+	resp, err := http.Get(url)
```

Comment:
We need to set the TLS config on the client for this outbound call.

Reply:
[Example](https://github.com/hyperledger/aries-framework-go/blob/4f10ba10077a9b0ef2e7a9531682549609cda2a4/test/bdd/pkg/util/util.go#L25-L31)

Reply:
Would need to add a new `vdr.ResolveOpts` option: https://github.com/hyperledger/aries-framework-go/blob/ca5e0650014e48021c33f0c96fe23a11123967cf/pkg/framework/aries/api/vdr/vdr.go#L65-L64

Reply:
Do you mean I should send the request from a https enabled client? I can add that but is there a preferred package or method that we use that I should use as a guideline?